hack/update/github.go

```diff
@@ -235,7 +235,12 @@ func GHReleases(ctx context.Context, owner, repo string) (stable, latest string,
 				if semver.Compare(ver, latest) == 1 {
 					latest = ver
 				}
+			} else if strings.Contains(prerls, "-alpha") {
```

Comment:
what about beta or rc? do we care about those?

Reply:
those are covered by latest above